de.ermshiperete.wine.BaseApp.yml

```diff
@@ -0,0 +1,124 @@
+id: de.ermshiperete.wine.BaseApp
+branch: 4.0
+
+runtime: org.freedesktop.Platform
+runtime-version: 1.6
```

Comment:
Perhaps you can do: `runtime-version: '18.08'` since `'1.6'` is going to be retired anytime soon. Also notice it's a string value.